docs/styles/docs/_code.scss

```diff
@@ -8,6 +8,8 @@
     font-family: var(--code-font);
     border-radius: 0.25rem;
     font-weight: 400;
+    // Create a new stacking context.
```

Comment:
It would also be good to capture in the comment _why_ it's important to create a new stacking context